consensus/misc/eip1559.go

```diff
@@ -101,18 +102,55 @@ func (f eip1559Calculator) CurrentFees(chainConfig *chain.Config, db kv.Getter)
 	return baseFee, blobFee, minBlobGasPrice, currentHeader.GasLimit, nil
 }
 
-// CalcBaseFee calculates the basefee of the header.
+// DecodeHolocene1599Params extracts the Holcene 1599 parameters from the encoded form:
```

Comment:
```suggestion
// DecodeHolocene1559Params extracts the Holcene 1559 parameters from the encoded form:
```

We need to fix 1599 to 1559 throughout the code.

Reply:
I fixed it throughout the whole code! :)